Mlem/App/Views/Shared/Navigation/NavigationPage+View.swift

```diff
@@ -132,10 +132,10 @@ extension NavigationPage {
                         .padding(.vertical, 6)
                         .background(.themedSecondaryGroupedBackground, in: .rect(cornerRadius: Constants.main.standardSpacing))
                 }
-                .disabled(instance.version < (minimumVersion ?? .zero))
+                .disabled(requiredFeature.map { !instance.software.supports($0) } ?? false)
             } header: {
-                if let minimumVersion {
-                    Text("This feature is only supported for instances running version \(String(describing: minimumVersion)) or later.")
+                if let requiredFeature {
+                    Text("This feature is only supported on instances running certain software versions.")
```

Comment:
Since we're losing precision, we might as well cut as many words as possible. I think the best way to handle this would be to have a (?) help tappable that opens a more detailed explanation, though that may be OOS here.

```suggestion
                    Text("This feature is not available on all instances.")
```

Reply:
Makes sense 👍 Yeah, I'm gonna leave that OOS and implement something like that down the line